README.md

```diff
@@ -159,21 +143,15 @@ Editing:
     the new file stem.  For example, the PDF corresponding to
     `source/reliability_average.Rmd` is `26-Chap-22_reliability_average.pdf`.
     Also see the chapter mappings at the end of this page.
-*   Rebuild the file you're working on from time to time with e.g.
-    `../scripts/rebuild_chapter reliability_average.Rmd` from the `source`
-    directory. Check the contents by opening e.g.
-    `../python-book/reliability_average.Rmd` (Quarto will show the correct
-    filename after the rebuild step).  If you get an error like this:
-
+*   Rebuild the file you're working on from time to time.
+    From the `source` directory:
     ```
-    Error in readLines(con, warn = readLines.warn) :
-      cannot open the connection
-    Calls: .main ... eval -> <Anonymous> -> yaml.load -> paste -> readLines
+    ninja ../python-book/reliability_average.html
     ```
-
-    then make sure you have run the `make set-version-python` or `make
-    set-version-r` command, to build the needed `_quarto.yml` and
-    `_variables.yml` files.
+    You can see the generated contents by opening e.g.
+    `../python-book/reliability_average.Rmd`
```

Comment:
Does the build actually generate `.Rmd` files?

Reply:
No, you're right, only HTML.